package.json

```diff
@@ -82,6 +83,7 @@
     }
   },
   "devDependencies": {
+    "@percy-io/percy-storybook": "^2.1.0",
     "@percy/puppeteer": "^0.4.0",
```

Comment:
Is this still needed?

Reply:
Nope. I'm going to do the big nuking at the end when I'm happy I've not missed anything.